code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm

```diff
@@ -5,6 +5,7 @@
 	icon = 'ModularTegustation/Teguicons/64x64.dmi'
 	icon_state = "melting_love"
 	icon_living = "melting_love"
+	dir = SOUTH
```

Comment:
This change does nothing, because dir is set on abnormality datum in respawn proc.